src/libsyntax_expand/lib.rs

```diff
@@ -33,6 +33,25 @@ pub use mbe::macro_rules::compile_declarative_macro;
 pub mod base;
 pub mod build;
 pub mod expand;
+#[macro_use] pub mod config;
 pub mod proc_macro;
 
 crate mod mbe;
+
+#[cfg(test)]
+mod tests;
+#[cfg(test)]
+mod parse {
+    mod tests;
```

Comment:
Until #64197 is fixed, `#[cfg]`s have to be placed directly on the out-of-line modules to avoid #61097.